pkg/kubectl/cmd/config/config_test.go

```diff
@@ -294,7 +294,7 @@ func TestEmbedNoKeyOrCertDisallowed(t *testing.T) {
 
 func TestEmptyTokenAndCertAllowed(t *testing.T) {
 	fakeCertFile, _ := ioutil.TempFile("", "cert-file")
-
+	defer os.RemoveAll(fakeCertFile.Name())
```

Comment:
I believe os.RemoveAll removes a directory, so os.Remove is the more normal choice.  But it turns out os.RemoveAll calls os.Remove first anyway, so ...